docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,252 @@
+# vCenter Event Broker Appliance - Troubleshooting
+
+## Requirements
+
+You must log on to the VEBA appliance as root. You can do this from the console. If you want SSH access, do the following:
+
+**Step 1** - After logging into the console as root, edit the sshd_config file
+
+```
+vi /etc/ssh/sshd_config
+```
+
+**Step 2** - Change the PermitRootLogin configuration line to `yes`. In vi, you can use the arrow keys to navigate up and down, delete characters with the `x` key, switch to insert mode with the `i` key, and then type your replacement characters. 
+
+```
+PermitRootLogin yes
+```
+
+**Step 3** - Save the configuration file. In vi, you hit the `Escape` key to get into command mode, type a colon `:` character, then `wq` for "write quit".
+
+```
+{Escape} :wq
+````
+
```

Comment:
Steps 1-3 isn't needed, root is already allowed to SSH but SSH Service is disabled by default. You just need to login via console and run `systemctl start sshd` (which will only turn on SSH but not enable it, so when you reboot, it'll be disabled again)

Reply:
Thanks, resolved.